src/emmy/numerical/unimin/brent.cljc

```diff
@@ -183,17 +189,19 @@
   ([f a b] (brent-min f a b {}))
   ([f a b {:keys [relative-threshold
                   absolute-threshold
+                  initial-guess
                   maxiter
                   maxfun
                   callback]
-           :or {relative-threshold (g/sqrt u/machine-epsilon)
-                absolute-threshold 1.0e-11
+           :or {relative-threshold brent-error
```

Comment:
I had flipped these relative to scmutils! Flipping them back is the right call imo even though it changes some test tolerances.